src/pages/ProductDesc/ProductDesc.js

```diff
@@ -1,8 +1,115 @@
 import React from "react";
 import "./ProductDesc.scss";
+import { useState, useEffect } from "react";
+import RecommendProducts from "./components/RecommendProducts/RecommendProducts";
+import NewProducts from "./components/NewProducts/NewProducts";
```

Comment:
import 순서에 맞게 수정해주세요!
```suggestion
import React, { useState, useEffect } from "react";
import RecommendProducts from "./components/RecommendProducts/RecommendProducts";
import NewProducts from "./components/NewProducts/NewProducts";
import "./ProductDesc.scss";
```